src/SmartTransactionsController.ts

```diff
@@ -521,8 +614,16 @@ export default class SmartTransactionsController extends BaseController<
           approvalTxFees,
           tradeTxFees,
         },
```

Comment:
you only want to set `fees` if it matches the networkClientId matches the current globally selected chainId, right?

Reply:
good catch! yessir!

Reply:
done here: https://github.com/MetaMask/smart-transactions-controller/pull/237/commits/caf5ac0a59b73823ef6977cb480bf4c3ec2b5b22